### PR TITLE
[bitnami/nginx] Adding sidecar support 

### PR DESCRIPTION
**Pull request by bvanderlugt**
**Description of the change**

Adding sidecar support to nginx

**Benefits**

The chart will support sidecars for deployments

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by kav** [comment]
I know @gajus found a different way for https://github.com/bitnami/charts/issues/4186 but we'd love this

File: bitnami/nginx/values.yaml [hunk 1]
```diff
@@ -467,6 +467,16 @@ ldapDaemon:
...
+## Sidecar paramaters
+##
+# sidecars:
+#   - name: your-image-name
+#     image: your-image
+#     imagePullPolicy: Always
+#     ports:
+#       - name: portname
+#         containerPort: 1234
+
```
Comment (dani8art) [inline bitnami/nginx/values.yaml:479]: We try to add in this way
```suggestion
sidecars:
```

**Review by dani8art — APPROVED** [review]
Hi! thank you so much for the PR, LGTM

Just a minor suggestion.

**Comment by marcosbc** [comment]
Thanks! I'm proceeding to merge

**Pull request merged**